ot/lp/__init__.py

```diff
@@ -349,6 +362,9 @@ def emd2(a, b, M, processes=multiprocessing.cpu_count(),
     center_dual: boolean, optional (default=True)
         If True, centers the dual potential using function
         :ref:`center_ot_dual`.
+    numThreads: int or "max", optional (default=1)
```

Comment:
Not clear, be clear that by defalt no OpenMP is used but if numThread>1 then attempt to use OpenMP if available